sycl/CMakeLists.txt

```diff
@@ -290,6 +290,7 @@ add_custom_target( sycl-toolchain
           llvm-objcopy
           sycl-post-link
           sycl-ls
+          xptifw
```

Comment:
I suggest we first check that project build is enabled before adding it to the toolchain.
I think we can't assume that all developers are using configure.py for configuring build system.